CHANGELOG.md

```diff
@@ -26,7 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
--
+- Removed deprecated classification code ([#1251](https://github.com/Lightning-AI/metrics/pull/1251))
```

Comment:
I see so let's make an extensive changelog to clearly mark what is the replacement of each removed class...

Reply:
should be better now, please take a look :)

Reply:
:+1: